design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -303,6 +303,58 @@ Cons:
 - Additional point of failure.
 - Additional latency when creating VMs with network binding plugins.
 
+##### Compute Container Resource Requirements / Limits
+
+For some plugins, such as passt, there is a need to execute an additional binary in the compute container.
+Since this binary has its own CPU and memory limits, they should be somehow accounted for.
```

Comment:
Please do not limit it to running an additional binary, the addition in memory or other resources may come from different reasons (e.g. libvirt itself requiring more memory due to the expected configuration).

Reply:
Done.